yb-voyager/src/query/queryissue/constants.go

```diff
@@ -434,28 +434,25 @@ Note: If the table is created as colocated, this hotspot concern can safely be i
-	LOW_CARDINALITY_DESCRIPTION_MULTI_COLUMN  = `The first column of this index has low cardinality. To improve data distribution, either convert it to a range-sharded index or reorder the columns so a higher-cardinality column appears first.
-Note: Range sharding is enabled by default only in PostgreSQL compatibility mode in YugabyteDB.`
+	LOW_CARDINALITY_DESCRIPTION               = "In distributed databases, index design should ensure even data distribution across multiple nodes. Indexes built on low-cardinality columns (e.g., boolean, days of the week) are not optimal, as they may lead to bad data distribution among tablets."
+	LOW_CARDINALITY_DESCRIPTION_SINGLE_COLUMN = `This index is built on a low-cardinality column. Refer to docs link for more details on recommendations.`
+	LOW_CARDINALITY_DESCRIPTION_MULTI_COLUMN  = `The first column of this index is low-cardinality column. Refer to docs link for more details on recommendations.`
 
-	NULL_VALUE_INDEXES_ISSUE_NAME                = "Null value indexes"
+	NULL_VALUE_INDEXES_ISSUE_NAME                = "Index on column with a high percentage of NULL values"
 	NULL_VALUE_INDEXES                           = "NULL_VALUE_INDEXES"
-	NULL_VALUE_INDEXES_DESCRIPTION               = "In distributed databases, index design should ensure even data distribution across nodes. Indexes on columns with many NULL values can lead to uneven distribution. Avoid indexing such columns directly."
-	NULL_VALUE_INDEXES_DESCRIPTION_SINGLE_COLUMN = `This index is built on a column having many NULLs. If NULLs are not queried, use a partial index to exclude them. If they must be indexed, consider combining this column with another column and create a range-sharded index to ensure better distribution.`
-	NULL_VALUE_INDEXES_DESCRIPTION_MULTI_COLUMN  = `The first column of this index has many NULLs and is not queried, use a partial index to exclude those rows. If NULLs must be indexed, convert the index to a range-sharded one.
-Note: Range sharding is enabled by default only in PostgreSQL compatibility mode in YugabyteDB.`
+	NULL_VALUE_INDEXES_DESCRIPTION               = "In distributed databases, index design should ensure even data distribution across nodes. Indexes on columns with many NULL values can lead to uneven distribution and can cause performance issues."
+	NULL_VALUE_INDEXES_DESCRIPTION_SINGLE_COLUMN = `This index is built on a column having high percentage of NULL values. Refer to docs link for more details on recommendations.`
+	NULL_VALUE_INDEXES_DESCRIPTION_MULTI_COLUMN  = `The first column of this index has high percentage of NULL values. Refer to docs link for more details on recommendations.`
 
-	MOST_FREQUENT_VALUE_INDEXES_ISSUE_NAME              = "Most frequent value indexes"
+	MOST_FREQUENT_VALUE_INDEXES_ISSUE_NAME              = "Index on column with high percentage of a value"
 	MOST_FREQUENT_VALUE_INDEXES                         = "MOST_FREQUENT_VALUE_INDEXES"
```

Comment:
can you change the type constants as well to be along the same lines of the new names we've chosen?